tests/symbolics/test_sympy_backend.py

```diff
@@ -35,6 +35,33 @@ def test_is_sympy_int(expression, expected):
     assert sympy_backend.is_constant_int(expression) == expected
 
 
+@pytest.mark.parametrize(
+    "expression_str, expected",
+    [
+        # Known constants
+        ("pi", pi),
+        ("PI", pi),
+        ("e", E),
+        ("E", E),
+        ("infinity", oo),
```

Comment:
Actually, I don't think we should have `infinity` here, as I don't even know in what cases it could arise. Any examples?